srcpkgs/gajim/template

```diff
@@ -1,24 +1,28 @@
 # Template file for 'gajim'
 pkgname=gajim
-version=1.3.3
+version=1.4.7
 revision=1
-wrksrc="gajim-gajim-${version}"
 build_style=python3-module
 hostmakedepends="gettext python3-setuptools"
-depends="python3-gobject python3-nbxmpp python3-setuptools python3-Pillow
- python3-cairo python3-precis-i18n python3-keyring python3-packaging
- python3-css-parser farstream python3-dbus python3-openssl"
+depends="python3-gobject python3-setuptools python3-Pillow
+ python3-cairo python3-dbus libsecret gupnp-igd gsound geoclue2 libnm
+ python3-nbxmpp python3-openssl python3-css-parser python3-keyring
+ python3-precis-i18n python3-packaging python3-setuptools python3-distro
+ python3-netifaces python3-pyasn1 libsoup farstream gst-plugins-base1
+ gst-plugins-ugly1 gst-libav gtksourceview4"
```

Comment:
where did you get this list of dependencies?

Reply:
It is a mixture from setup.cfg and archlinux pkg and https://dev.gajim.org/gajim/gajim/. Did i miss something?